frame/contracts/src/wasm/prepare.rs

```diff
@@ -439,7 +433,7 @@ fn do_preparation<C: ImportSatisfyCheck, T: Config>(
 		schedule,
 	)?;
 	Ok(PrefabWasmModule {
-		schedule_version: schedule.version,
+		schedule_version: schedule.instruction_weights.version,
```

Comment:
I think `PrefabWasmModule::schedule_version` should be renamed to `instruction_weights_version` (or something shorter) to make it more readable

Reply:
Renamed.